### PR TITLE
Fixed ignored sites in firefox

### PR DESCRIPTION
**Pull request by JeroenED**
Because I was freaking out on issue #165 I tried find a solution. Appeared to be the most simple fix ever.

Here is the PR for it :)

Fixes https://github.com/keepassxreboot/keepassxc-browser/issues/165.

### CODE REVIEW

File: keepassxc-browser/keepassxc-browser.js [hunk 1]
```diff
@@ -1559,7 +1559,7 @@ cip.initCredentialFields = function(forceCall) {
         cip.initializeSitePreferences();
         if (cip.settings.sitePreferences) {
             for (const site of cip.settings.sitePreferences) {
-                if (site.url === document.location.href || siteMatch(site.url, document.location.href)) {
+                if (site.url === document.location.href || siteMatch(site.url, window.top.location.href)) {
```
Comment (droidmonkey) [inline keepassxc-browser/keepassxc-browser.js]: The only time ```document.location.href``` and ```window.top.location.href``` would be different is in a frames scenario. In this case, is this the desired behavior? You would also want to change the other half of the if statement if it is.

Comment (JeroenED) [reply keepassxc-browser/keepassxc-browser.js]: Yes, this is indeed the required behavior. #165 handles about wysiwyg editors who add additional data-cip-id attributes to input fields. They use iframes which firefox sees as a frame with a different document.location (which is actually a empty string) Chrome/chromium has different behavior and was already solved. This fix does not change anything to chromium.

Comment (droidmonkey) [reply keepassxc-browser/keepassxc-browser.js]: Great!

**Comment by varjolintu** [comment]
Excellent! Thank you.

**Comment by JeroenED** [comment]
Can you give some clear instructions on how repack the XPI? I know the fix, I can redownload the xpi, I can repack the xpi, but I can't seem to install the xpi. I get a corrupted package error.

(Yes, it's a very annoying issue to me 😉)

**Comment by varjolintu** [comment]
Just zip it. Or you can load the extension from a path using about:debugging (load the manifest.json file through it).

**Pull request merged**